### PR TITLE
Dev video

### PR DESCRIPTION
**Pull request by GustavJaner**
Video playback from S3 bucket possible

### CODE REVIEW

File: pages/S3_handler.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const secrets = require('../secrets.json');
+
+const credentials = {
+  accessKeyId:     secrets.ACCESS_KEY_ID,
+  secretAccessKey: secrets.SECRET_KEY_ID,
+  region: 'eu-central-1',
+};
+
+var AWS = require('aws-sdk');
```
Comment (edgarlpassos) [inline pages/S3_handler.jsx]: Put all `require` statements at the top of the file.

**Comment by edgarlpassos** [comment]
Do we need the stuff that is in `static/`? It looks like we're getting all of those from S3

**Comment by GustavJaner** [comment]
All files in static is just left from some tests from previous, so yes I'll remove it

File: pages/S3_handler.jsx (hunk 1 above)
Comment (GustavJaner) [reply pages/S3_handler.jsx]: Ofc!

**Pull request closed without merging**